src/main/notifications/dnd-windows.ts

```diff
@@ -20,13 +20,9 @@ function getWindowsDoNotDisturb() {
     case 2:
         return true;
     case 1:
-        return !isPriority('Mattermost.Desktop');
-    case 0:
-    case -1:
-    case -2:
-        return false;
+        return !(isPriority('Mattermost.Desktop').value);
```

Comment:
So this turned out to be the right condition to check but we were not reading the right value?

Reply:
Correct

Reply:
Truthy/falsy values strike again!